spec/outcome_spec.rb

```diff
@@ -0,0 +1,61 @@
+describe Outcome do
+
+  it "should check for all types of horizontal wins" do
+    current_game = Outcome.new
```

Comment:
Considering being DRY in your tests. I've noticed that you're constantly constructing a new `Outcome`. You should consider looking into before hooks to avoid being repetitious: https://relishapp.com/rspec/rspec-core/v/2-0/docs/hooks/before-and-after-hooks

Reply:
Thanks for the tip! I used it for the Player spec and forgot all about it. Done!